actions/milab-shell-utils/action.yaml

```diff
@@ -0,0 +1,12 @@
+name: MiLab various shell utilities
+author: 'MiLaboratories'
+description: |
+  Add to PATH various MiLaboratories shell utilities.
```

Comment:
```
Brings scripts inside this action into workflow and makes them available to be called by name, by modifying PATH environment variable.
```

Reply:
You description is much better and clear. Thank you.